requirements.txt

```diff
@@ -0,0 +1 @@
+awscli==1.7.25
```

Comment:
I believe Heroku detects the stack based on whether it sees `requirements.txt` or `Gemfile`. So if it sees `requirements.txt` then it will treat the app as a python app.